qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy

```diff
@@ -442,6 +492,13 @@ oidc:
                 )
                 .getAuthProvidersCount() == 0
 
+        // No notifier should be created.
+        def notifierAfterDeletion = NotifierService.getNotifierClient().getNotifiers(
```

Comment:
Hmm, why all results here are named `...AfterDeletion` when in reality they weren't created yet?